orchagent/portsorch.cpp

```diff
@@ -6163,6 +6163,10 @@ void PortsOrch::generateQueueMapPerPort(const Port& port, FlexCounterQueueStates
         queueVector.emplace_back(name.str(), id);
         if (voq)
         {
+            // Install a flex counter for this voq to track stats. Voq counters do
+            // not have buffer queue config. So it does not get enabled through the
```

Comment:
FYI, We do apply egress_lossy_profile/egress_lossless_profile on VoQs via SONIC config. This change is there in master branch.

Reply:
@vmittal-msft https://github.com/sonic-net/sonic-swss/blob/f66abeda6a6e67b40b997b779ec3a6a362632fc1/orchagent/flexcounterorch.cpp#L331 still needs to enhanced to parse config for sysports. Till them we cannot conditionally enable voq counters.

Reply:
One clarification, Will parsing fix egress queue counter fetching or VoQ counters ? Based on this, we may/may not revert your "always enable" logic.